src/test/java/io/vertx/ext/mongo/MongoClientTest.java

```diff
@@ -77,6 +77,27 @@ public void testFindBatchResumePause() throws Exception {
     });
   }
 
+  @Test
+  public void findBatchResumePause() throws Exception {
+    int docs= 111;
```

Comment:
whitespace

Reply:
docs --> numberOfDocs or docsCount or something like that

Reply:
make variable names great again